packages/packages/v1-adapters/src/hooks/use-is-route-active.ts

```diff
@@ -0,0 +1,28 @@
+import { useEffect, useState } from 'react';
+import {
+	isRouteActive,
+	listenTo,
+	routeCloseEvent,
+	routeOpenEvent,
+	RouteEventDescriptor,
+} from '../';
+
+export function useIsRouteActive( route: RouteEventDescriptor['name'] ): boolean {
+	const [ isActive, setIsActive ] = useState( () => isRouteActive( route ) );
```

Comment:
The route status should be re-calculated when the route is changed.

Reply:
Done